tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
```

Comment:
Needed this to get jest to be happy with `import React from 'react'` (even though `import * as React from 'react'` is more correct).